server/middleware/authenticator.js

```diff
@@ -19,8 +19,10 @@ const authenticator = {
         if (user) {
```

Comment:
How did you fix this issue

Reply:
I traced it back to the model that gets users from the database. It returned an array instead of an object. So I fixed that.